### PR TITLE
added the ability to control the "incremental" flag from command and api

### PR DESCRIPTION
**Pull request by arossert**
Following #145, this PR allows the user to control the "incremental" flag both in the command and API.
When setting the incremental flag to 1 it fixes the capture on the latest `0.12.1` version.

### CODE REVIEW

File: vncdotool/command.py [hunk 1]
```diff
@@ -172,7 +172,7 @@ def build_command_list(factory, args, delay=None, warp=1.0):
                         imgformat, SUPPORTED_FORMATS), file=sys.stderr)
                 sys.exit(1)
             else:
-                factory.deferred.addCallback(client.captureScreen, filename)
+                factory.deferred.addCallback(client.captureScreen, filename, int(incremental_refreshes))
```
Comment (sibson) [inline vncdotool/command.py]: The type conversion here feels a bit strange.  Can we avoid this somehow, do it earlier, later or trust duck typing to do the right thing?

File: vncdotool/command.py [hunk 2]
```diff
@@ -398,6 +398,9 @@ def vncdo():
         metavar='FACTOR', default=1.0,
         help='pause time is accelerated by FACTOR [x%default]')
 
+    op.add_option('-i', '--incremental-refreshes', action='store_true', default=False,
+        help='set the "incremental" flag')
```
Comment (sibson) [inline vncdotool/command.py]: I always struggle with these flag names.  I'd like to make them more approachable to users and hide VNC protocol details but that can also make it harder to understand.  I guess my real question is what is the correct default, what is the most compatible setting we can use that works for most people out of the box?

Comment (arossert) [reply vncdotool/command.py]: The current default value is 0 (False) so I kept it the same.
Regarding the flag name, I think that it needs to give enough information on what it is used for, I can change it if you have a better idea.

File: vncdotool/command.py (hunk 1 above)
Comment (arossert) [reply vncdotool/command.py]: The function expecting an int value and the flag is boolean, do you think that we need to convert it before?

**Comment by sibson** [comment]
thanks for putting this together.

**Pull request merged**